app/jsx/components/DrawerComp.jsx

```diff
@@ -78,23 +92,23 @@ class SortableNavList extends React.Component {
         scaffoldBlockPxWidth={30}
         maxDepth={2}
         canDrag={({ node }) => {
-          if (node.type == "home") // don't allow dragging unit home
+          if (node.type.includes("fixed")) // don't allow dragging unit home or journal issues drop down
             return false
           let slug = (node.type == "page") ? node.slug : node.type
           if (!(this.props.cms.permissions.nav_perms[slug] || {}).reorder)
             return false
           return true
         }}
         canDrop={({ node, nextTreeIndex, nextParent }) => {
-          if (this.state.firstIsHome && nextTreeIndex == 0) // don't allow rearranging above the unit home
+          if (this.state.firstIsFixed && nextTreeIndex <= 0) // don't allow rearranging above fixed nodes
             return false
           if (!nextParent)
             return true
-          if (node.type == "folder")
+          if (node.type.includes("folder"))
```

Comment:
Why the change from == "folder" to includes("folder")? Are there nodes of some larger string which contains "folder" that need to be treated as folders? Maybe this question will be answered as I continue reading...